app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -31,7 +31,6 @@ def plan(capsule_content, options = {})
               end
               plan_action(ConfigureCapsule, capsule_content, environment, content_view, repository)
               sync_repos_to_capsule(capsule_content, repository_ids)
-              plan_action(RemoveOrphans, :capsule_id => capsule_content.capsule.id)
```

Comment:
We'd still need this for non-yum and non-puppet content.

I do think we might should move this to the 'delete_orphaned_content' rake task, and tie into foreman-tasks for both  the server and capsules.  Then this could be removed.